org.eclipse.transformer/src/main/java/org/eclipse/transformer/Transformer.java

```diff
@@ -707,20 +709,40 @@ protected UTF8Properties loadProperties(AppOption ruleOption, Set<String> orphan
 	}
 
 	private String relativize(String relativeRef, String baseRef) {
-		int lastSlash = baseRef.lastIndexOf('/');
-		if (lastSlash == -1) {
-			return relativeRef;
-		} else {
-			return baseRef.substring(0, lastSlash + 1) + relativeRef;
-		}
+		Path basePath = FileSystems.getDefault().getPath(baseRef);
```

Comment:
This should be `Path basePath = Paths.get(baseRef)`. We should avoid going to `FileSystems` unnecessarily.

Reply:
Hi; updated.